### ISSUE TITLE
Accessibility :MAS13: Zoom: RHS pane contents are missing when zoomed to 200%

### ISSUE THREAD

**Issue by wiproaccess** [wiproaccess/vscode-website#23]
#A11yMAS
#MAS13
#Wipro
#vscode-website

**Testing Environment:**
OS-RS1
OS Name Microsoft -Windows 10 Pro
Browser-Edge
Edge Version-Microsoft Edge 38.14393.461.0

**Repro Steps:**
1.Open https://code.visualstudio.com/docs in edge,zoom the page to 200%.
2.Navigate to the RHS pane beside the "Getting Started" below the header of the page.
3.Observe the RHS pane.[screenshot]

**Actual Result:**
Observe the contents at  RHS pane of the page are missing when zoomed to 200% but clearly visible when zoomed to normal (100%).
**Issue repro in below resolutions:
Resolution: 1366x768
Resolution: 1280x1024**

**Note**: Missing contents are 
The contents under **"Getting Started"** and social links  "Subscribe", "Ask questions", "Follow @code", "Request features", "Report issues".[screenshot]

**Expected Result:**
The contents should not be missed when zoomed to 200%.

**User Impact:**
Users with low vision and people who need cognitive assistance will not be able identify the missing interactive elements.

**Screenshots:**
![missing rhs pane contents when zoomed to 200prcnt](https://cloud.githubusercontent.com/assets/24825441/21709307/07e03e6c-d406-11e6-9c34-c8b54495ec71.png)
![visible contents in rhs pane_normal view](https://cloud.githubusercontent.com/assets/24825441/21709309/0aec8020-d406-11e6-8855-0a1366e8dedc.png)

**Comment by chrisdias** [comment]
The RHS "menu" (e.g. GETTING STARTED... VS Code in Action... etc) are landmarks as to where you are in the page. There is no loss of functionality, these simply scroll to that section in the document. 

The fact that the Social links (e.g. Subscribe, Ask Questions, etc.) are missing would be a bigger problem, but these are still reachable through the new [Community](https://code.visualstudio.com/community) page, which is reachable from the hamburger at the top of the page (when zoomed in to 200%).
